content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -214,6 +214,10 @@ The `from-key` value is specific to the integration type, and should exclude the
 `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
 : When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, this regex redacts sensitive data from incoming requests' query string reported in the `http.url` tag (matches are replaced with `<redacted>`). This regex executes for every incoming request.
 
+`DD_INSTRUMENTATION_TELEMETRY_ENABLED`
+Datadog may collect [environmental and diagnostic information about your system][15] to improve the product. When false, this telemetry data will not be collected.<br>
```

Comment:
```suggestion
: Datadog may collect [environmental and diagnostic information about your system][15] to improve the product. When false, this telemetry data will not be collected.<br>
```

Reply:
It looks like this is being rendered outside of the "box" (see the staging preview). I think if you add the colon here, it will fix the issue.